packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -463,7 +468,7 @@ describe('Transfer tests', function(this: any) {
         before(`start geth on sync: ${syncMode}`, () => startSyncNode(syncMode))
 
         describe('Transfer CeloGold >', () => {
-          const GOLD_TRANSACTION_GAS_COST = 30005
+          const GOLD_TRANSACTION_GAS_COST = 21000
```

Comment:
nit: Consider `INTRINSIC_TX_GAS_COST`

Reply:
And instead of 155000 elsewhere, consider defining an `ADDITIONAL_INTRINSIC_TX_GAS_COST = 134000` and setting expectedGas everywhere to be either `INTRINSIC_TX_GAS_COST` or `INTRINSIC_TX_GAS_COST + ADDITIONAL_INTRINSIC_TX_GAS_COST`

Reply:
good call